integration/simulation/network/network.go

```diff
@@ -31,9 +33,10 @@ type RPCHandles struct {
 	// an obscuro client per obscuro node in the network (used for things like validation rather than transactions on behalf of sim accounts)
 	ObscuroClients []rpcclientlib.Client
 
-	// an RPC client per wallet, with a viewing key setup (on the client and registered on its corresponding host enclave),
+	// an RPC client per node per wallet, with a viewing key setup (on the client and registered on its corresponding host enclave),
 	//	to mimic user acc interaction via a wallet extension
-	VirtualWalletExtensionClients map[string]rpcclientlib.Client // an obscuro client per wallet (configured with viewing key where applicable)
+	// map of address hex string -> slice of clients (indexed by node)
```

Comment:
@BedrockSquirrel Not sure what this means. Can you clarify? The keys are the hex strings of whose/which addresses? Which clients are in the value slice? Should it be:

`// map of owner addresses to RPC clients for that owner (one RPC client per L2 node)`